woocommerce/woocommerce
# Remove mariaDB image tag workaround

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





#44841 introduced a workaround to force 11.2 tag for mariadb Docker image used in wp-env. This was caused by an upstream issue with the mariadb images out of our control.
A fix is now in place and we can revert the workaround to use again the default version (latest).



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

- Locally, remove `plugins/woocommerce/node_modules` and `plugins/woocommerce-blocks/node_modules` folders, then run `pnpm install`. Following a successful install, run `pnpm env:start` in any of the plugins root. No errors are expected, the test environment should run.
- In CI all tests should run and pass.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
